components/todo-input.js

```diff
@@ -1,20 +1,110 @@
-import React from "react";
+import React, { useState } from 'react';
+
 import styled from "styled-components";
 
-export default function TodoInput() {
+export default function TodoInput({addTodo}){
+
+  const [newTodo, setNewTodo] = useState({});
+
+  const changeInputData = (e) => {
+    setNewTodo({...newTodo, [e.target.name]:e.target.value});
+  }
+
   return (
-    <Wrapper>
-      리액트-투두
-      <br />
-      시간을 입력하는 칸
-      <br />
-      Todo를 입력하는 칸
-    </Wrapper>
+    <InputWrapper>
+      <TimeBox>
+        <Time>시간</Time>
+        <InputDay name="date" type="number" placeholder="날짜를 입력하세요 (ex.20200404)" 
+        onInput = {(e) =>{e.target.value = Math.max(0, parseInt(e.target.value) ).toString().slice(0,8)}}
+        onChange={changeInputData}
+        />
+      </TimeBox>
+      <TodoBox>
+        <Todo>TODO</Todo>
+        <InputTodo name="content" placeholder='할 일을 입력하세요' onChange={changeInputData}></InputTodo>
+      </TodoBox>
+      <EnterBtn onClick={() => addTodo({...newTodo})}>등록</EnterBtn>
+    </InputWrapper>
```

Comment:
```suggestion
    <Wrapper>
      <Row>
        <Label>시간</Label>
        <Input name="date" type="number" placeholder="날짜를 입력하세요 (ex.20200404)" 
          onChange={handleFormChange}
        />
      </Row>
      <Row>
        <Label>TODO</Label>
        <Textarea name="content" placeholder='할 일을 입력하세요' onChange={handleFormChange}/>
      </Row>
      <SubmitButton onClick={() => onSubmit(newTodo)}>등록</SubmitButton>
    </Wrapper>
```
1. 컴포넌트 이름이 바뀐 것들은 아래 styled-component 정의 부분에 달린 리뷰들을 참고해주세요.
2. onInput 문법이 존재하긴 하지만 리액트 팀에서는 이것의 사용을 추천하지 않고 있습니다. [참고](https://thebook.io/006961/part01/ch07/01/01-01/)
onChange가 있는데 onInput을 또 써줄 필요는 없으므로 삭제해주시면 됩니다.
3. onClick에 함수 잘 넣어주셨어요 👍